src/api/objects/LegacyObjectAPIInterceptor.js

```diff
@@ -1,15 +1,56 @@
 define([
     './object-utils',
-    './ObjectAPI'
+    './ObjectAPI',
+    './objectEventEmitter'
 ], function (
     utils,
-    ObjectAPI
+    ObjectAPI,
+    objectEventEmitter
 ) {
-    function ObjectServiceProvider(objectService, instantiate) {
+    function ObjectServiceProvider(objectService, instantiate, topic) {
         this.objectService = objectService;
         this.instantiate = instantiate;
+
+        this.topicService = topic;
+        this.generalTopic = topic('mutation');
+        this.bridgeEventBuses();
     }
 
+    /**
+     * Bridges old and new style mutation events to provide compatibility between the two APIs
+     * @private
+     */
+    ObjectServiceProvider.prototype.bridgeEventBuses = function () {
+        var generalTopicListener;
+
+        var handleMutation = function (newStyleObject) {
+            var keyString = utils.makeKeyString(newStyleObject.key);
+            var oldStyleObject = this.instantiate(utils.toOldFormat(newStyleObject), keyString);
+            var specificTopic = this.topicService("mutation:" + keyString);
+
+            // Don't trigger self
+            if (generalTopicListener){
+                generalTopicListener();
+            }
+            this.generalTopic.notify(oldStyleObject);
+            specificTopic.notify(oldStyleObject.getModel());
```

Comment:
Sorry for the confusion, I thought that was commonly accepted: Old views must listen to mutation, otherwise they are expected to fall out of sync.  

I think that in most cases this is already true, so we don't have anything to do. But if we see out of sync problems we should be able to handle them relatively easily.  At the same time, this ties the mutation listener lifecycle to the view; we don't need to synchronize models that are not currently visible to the end user.